platform/src/main/java/org/stellar/anchor/platform/rpc/RequestOnchainFundsHandler.java

```diff
@@ -140,9 +140,6 @@ protected void validate(JdbcSepTransaction txn, RequestOnchainFundsRequest reque
     if (request.getAmountIn() == null && txn.getAmountIn() == null) {
       throw new InvalidParamsException("amount_in is required");
     }
-    if (request.getAmountOut() == null && txn.getAmountOut() == null) {
-      throw new InvalidParamsException("amount_out is required");
-    }
```

Comment:
Shouldn't we require `amount_out` when a quote was used or when `amount_in_asset == amount_out_asset`?

Reply:
Good point. I think we should require amount_out if there was no quote as well

Reply:
I don't think we can require `amount_out` if there was no quote, since the business doesn't have to commit to a payout amount until its about to send funds. The exception is if `amount_out_asset = amount_in_asset`, since its always 1:1 minus fees.

Reply:
Sorry we need to require either amount_out to be present either in request or in transaction